src/adjoint/DASolver/DASolver.C

```diff
@@ -510,6 +505,9 @@ void DASolver::getForcesInternal(
     tmp<volSymmTensorField> tdevRhoReff = daTurb.devRhoReff();
     const volSymmTensorField::Boundary& devRhoReffb = tdevRhoReff().boundaryField();
 
+    const pointMesh& pMesh = pointMesh::New(meshPtr_());
+    const pointBoundaryMesh& boundaryMesh = pMesh.boundary();
```

Comment:
I added this reference so that we can get the number of points per patch. Is that accessible from any other mesh object that already exists here so we do not need to generate a new one?